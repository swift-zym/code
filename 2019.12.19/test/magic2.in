10 10
hehhdeaaeh
239 323 386 757 591 81 920 70 494 861
hhdeaae
eaaeh
a
eh
a
hehh
ehh
deaaeh
he
h
