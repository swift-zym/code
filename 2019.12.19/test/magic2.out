find hhdeaae at 8
find eaaeh at 9
find a at 6
find a at 7
find eh at 2
find eh at 9
find a at 6
find a at 7
find hehh at 3
find ehh at 3
find deaaeh at 9
find he at 1
find h at 0
find h at 2
find h at 3
find h at 9
ok 1
ok 3
ok 4
ok 5
ok 6
ok 8
ok 9
ok 10
4050
